### ISSUE TITLE
The preview view show incorrect ratio after stop video in landscape orientation

### ISSUE THREAD

**Issue by jcsofts** [CameraKit/camerakit-android#494]
### Is this a bug report?

Yes

### Have you read the [Contributing Guidelines](.github/CONTRIBUTING.md)?

Yes

### Steps to Reproduce

(Write your steps here:)
I test the demo of camerakit. 

1. first hold down the the video button in landscape orientation
2. release the video button to stop the video
3. the preview will be scale distortion

### Expected Behavior

preview will be scale distortion

### Actual Behavior

(Write what happened. Add screenshots!)

### Reproducible Demo

(Paste the link to an example project and exact instructions to reproduce the issue.)

**Comment by hksfho** [comment]
same as you

**Comment by tyoc213** [comment]
same on some phones

**Comment by ErAmrit** [comment]
Yes, I am also experiencing the same issue, there is any clue to fix it.

**Comment by helang1991** [comment]
me too

**Comment by FuherProgramer** [comment]
me too ! but why no body answer us and fix this big issue
